### ISSUE TITLE
Screenshot created but not copied to clipboard

### ISSUE THREAD

**Issue by Justtaway** [ObserverOfTime/mpv-scripts#13]
### Script(s)

`clipshot.lua`

### Description

Image is created in temp folder, but doesn't go to clipboard.
I have Win11x64 22631.3296.

I tried enabling logs and got this, the most relevant line, as I believe, is:
`[  15.820][e][cplayer] Starting subprocess failed: init
`
Here is full logs:
[output.txt](https://github.com/ObserverOfTime/mpv-scripts/files/14859615/output.txt)

I tried another script at https://github.com/zc62/mpv-scripts/blob/master/screenshot-to-clipboard.js and it does work.

### Possible Fix

-

### Steps to Reproduce

1. Open video file
2. Press any clipshot hotkey (c, C, Alt+c)
3. See OSD message about screenshot being copeid to clipboard
4. Check clipboard (via Clipdiary, Windows stock clipboard manager or just pasting into any app)
Expected result: screenshot can be pasted from clipboard to any app
Actual result: there is no screenshot in clipboard, but file is created in temp folder. Tried changing jpeg to png, same result.

### Operating System(s)

- [X] Windows
- [ ] Linux
- [ ] macOS

### MPV version

 mpv.net 7.1.1.0 (03.02.2024), mpv v0.37.0-152-gbd5d8e41 (11.01.2024)

### Lua version

5.4.4

**Comment by ObserverOfTime** [comment]
This is the command that the script tries to run on Windows:
```
"powershell" "-NoProfile" "-Command", "& {
            Add-Type -Assembly System.Windows.Forms;
            Add-Type -Assembly System.Drawing;
            $shot = [Drawing.Image]::FromFile(\"C:\\\\Users\\\\cuzzo\\\\AppData\\\\Local\\\\Temp\\\\mpv-screenshot.jpeg\");
            [Windows.Forms.Clipboard]::SetImage($shot);
        }"
```
Can you run it (either in cmd or powershell) and see if there's an error?

**Comment by Justtaway** [comment]
> Can you run it (either in cmd or powershell) and see if there's an error?

How do I do it properly? Open mpv, copypaste enitre script in powershell? Or line by line?

**Comment by ObserverOfTime** [comment]
Just open powershell and copy paste the whole thing (assuming the file is still there).

**Comment by ObserverOfTime** [comment]
At least translate it…

**Comment by Justtaway** [comment]
PS C:\Users\cuzzo> "powershell" "-NoProfile" "-Command" "& {
>> Add-Type -Assembly System.Windows.Forms;
>> Add-Type -Assembly System.Drawing;
>> $shot = [Drawing.Image]::FromFile(\"C:\\\\Users\\\\cuzzo\\\\AppData\\\\Local\\\\Temp\\\\mpv-screenshot.jpeg\");
>> [Windows.Forms.Clipboard]::SetImage($shot);
>> }"
string:1 character:14
+ "powershell" "-NoProfile" "-Command" "& {
+ ~~~~~~~~~~~~
The unexpected token ""-NoProfile"" in an expression or operator.
line:1 character:27
+ "powershell" "-NoProfile" "-Command" "& {
+ ~~~~~~~~~~
The unexpected token ""-Command"" in an expression or operator.
line:1 character:38
+ "powershell" "-NoProfile" "-Command" "& {
+ ~~~~
Unexpected token ""&{
Add-Type -Assembly System.Windows.Forms;
Add-Type -Assembly System.Drawing;
$shot = [Drawing.Image]::fromFile(\"" in an expression or operator.
line:4 sign:49
+ ... :FromFile(\"C:\\\\Users\\\\cuzzo\\\\AppData\\\\Local\\\\Temp\\\\mpv-s ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
An unexpected token "C:\\\\Users\\\\cuzzo\\\\AppData\\\\Local\\\\Temp\\\\mpv-screenshot .jpeg\");
[Windows.Forms.Clipboard]::SetImage($shot);
}"" in an expression or operator.
+ CategoryInfo : ParserError: (:) [], ParentContainsErrorRecordException
+ FullyQualifiedErrorId : UnexpectedToken

**Comment by ObserverOfTime** [comment]
Does it work if you run it from cmd instead?

**Comment by Justtaway** [comment]
No, cmd can't even recognize those commands.

**Comment by ObserverOfTime** [comment]
I could not reproduce the error in my VM but it should be fixed by 31c31b7.
